src/app/story/storyrow/storyrow.component.html

```diff
@@ -1,11 +1,16 @@
 <div class="node-uid-column story-row-column">
-  <div class="node-display-name">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
-  <div class="node-timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">
-    {{ node.timeline_timestamp | date: 'MMM d, y, HH:mm:ss':'+0000' }} UTC
+  <div class="node-uid-column__display-name lengthened">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
+  <div class="node-uid-column__display-name shortened">{{ node.display_name | nodeDisplayName | truncate: 15 }}</div>
+  <div class="node-uid-column__timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">
+    {{ node.timeline_timestamp | date: 'MMM d, y, HH:mm:ss':'+0000' }} <div class="timezone">UTC</div>
```

Comment:
The div should be a span because you just want to apply styles to it and don't need display attributes.